src/index.css

```diff
@@ -2,13 +2,29 @@
 @import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300..700&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
 
 :root {
+   --bg-primary: #e9ecef;
+   --border-color: #ccc;
+   --shadow-color: rgba(0, 0, 0, 0.2);
+   --shadow-color-hover: rgba(0, 0, 0, 0.3);
+   --title-color: #333;
+   --button-bg-color: #4dabf7;
+   --button-bg-color-hover: #228be6;
+   --description-color: #2c3e50;
+   --target-color: #34495e;
+   --progress-low: #f08c00;
+   --progress-low-hover: #e8590c;
+   --progress-medium: #ffd43b;
+   --progress-medium-hover: #fab005;
+   --progress-high: #69db7c;
+   --progress-high-hover: #40c057;
```

Comment:
I might have not been clear when we talked about it, but these variables should be in a file that its whole purpose is adding variables, and not here, so it would be easy to maintain and use and with only one purpose of declaring variables.

Like we did in the duckfooding project.

So the variables are a kind of an asset for the whole project so it should be there.
The file can be called ```variable.module.css``` or ```theme.module.css``` or such.
You can decide if it needs to be under a folder or not (not critical imo in this project), like would it be ```assets/variables.module.css``` or ```assets/design/variables.module.css```.